### PR TITLE
Add appmodule for 64-bit version of Media Player Classic

### LINKED ISSUE #6066: support for Media Player Classic 64 bit
Greetings.
I like the fact that NVDA has support for Media Player Classic. This support does not however transfer to the 64 bit version. Funny enough, I investigated the matter. In the library.zip folder, if I copied the mpc-hc.pyo file and renamed it mpc-hc64.pyo based on the application name, it works equally well with NVDA, supporting the status bar and other features. Can this be done to the main distribution?

### PR DESCRIPTION
**Pull request by lukaszgo1**
### Link to issue number:
Fixes  #6066
### Summary of the issue:
NVDA has an app module for MPC-HC, but only for 32-bit version.

### Description of how this pull request fixes the issue:
I've created an app module for 64-bit version of MPC-HC which simply uses a 32-bit app module.
### Testing performed:
Tested that status bar is properly recognized in 64--bit version of MPC-HC and that app module is loading.
### Known issues with pull request:
None

### Change log entry:
 * Bug fixes
The 64-bit version of Media Player Classic is now supported.

### CODE REVIEW

File: source/appModules/mpc-hc64.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2018 NV Access Limited, Łukasz Golonka
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+from mplayerc import *
```
Comment (leonardder) [inline source/appModules/mpc-hc64.py]: Could you please add an empty line before and after this line?

**Review by LeonarddeR — DISMISSED** [review]

File: source/appModules/mpc-hc64.py (hunk 1 above)
Comment (lukaszgo1) [reply source/appModules/mpc-hc64.py]: @leonardder Done

**Comment by lukaszgo1** [comment]
@leonardder I believe I've addressed your commends. Is your approval enough to incubate this or something else needs to be done?

**Comment by leonardder** [comment]
Yes, I think it is safe to incubate this, as it is a very small change. Could you please merge in most recent master before this will be merged into master in around a week or two?

**Review by michaelDCurran — DISMISSED** [review]

**Pull request merged**